source/tutorials/nvidia.rst

```diff
@@ -26,17 +26,37 @@ require a manual installation.
    be required.
 
 .. contents:: :local:
-    :depth: 1
+    :depth: 2
```

Comment:
Revert to depth of 1 please.

Reply:
I did this intentionally. The way this article gets used/referenced, having the L2 sections visible and accessible helps. Even the sidebar is collapsed by default.

Reply:
In general,  internal TOCs are set to depth 1 for tutorials. Please keep the original setting.